llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -973,11 +974,13 @@ void VPWidenIntrinsicRecipe::execute(VPTransformState &State) {
 
   CallInst *V = State.Builder.CreateCall(VectorF, Args, OpBundles);
 
+  // FIXME: vp.cast and vp.select dont pass the underlying instruction into the
```

Comment:
This comment is confusing here I think, it should be at the place where it is not passed, as this needs to be fixed where the recipe is constructed, right?

Reply:
Removed. I will try to add Flags during the recipe construction with new patch